must-gather/gather-olm-info

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
all shell script should have `set -eu -o pipefail` to ensure that the script is robust

Reply:
done